schemas/org.whatwg/media_element/jsonschema/1-0-0

```diff
@@ -0,0 +1,274 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Context Schema for a media player event",
+  "self": {
+    "format": "jsonschema",
+    "name": "media_element",
+    "vendor": "org.whatwg",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "autoPlay": {
+      "type": "boolean",
+      "description": "If playback should automatically begin as soon as enough media is available to do so without interruption"
+    },
+    "buffered": {
+      "description": "An array of time ranges that have been buffered",
+      "items": {
+        "type": "object",
+        "description": "A time range object",
+        "properties": {
+          "start": {
+            "description": "The beginning of the time range",
+            "maximum": 9007199254740991,
+            "minimum": 0,
+            "type": "number"
+          },
+          "end": {
+            "description": "The end of the time range",
+            "maximum": 9007199254740991,
+            "minimum": 0,
+            "type": "number"
+          }
+        },
```

Comment:
Is a 0-length duration valid if the lower end is not `0`? E.g., is it valid to have a buffered duration from time unit `10` to time unit `10`?

Reply:
It looks like 0-length durations are valid:
["The start of a range must be less than or equal to the end of that same range."](https://html.spec.whatwg.org/multipage/media.html#timeranges)